examples/taskruns/sidecar-ready-script.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: tekton.dev/v1alpha1
+kind: TaskRun
+metadata:
+  generateName: sidecar-ready-
+spec:
+  taskSpec:
+    sidecars:
+    - name: slow-sidecar
+      image: ubuntu
+      script: |
+        echo "hello from sidecar" > /shared/message
+      volumeMounts:
+      - name: shared
+        mountPath: /shared
+
+    steps:
+    - name: check-ready
+      image: ubuntu
+      # The step will only succeed if the sidecar has written this file, which
+      # it does 5s after it starts, before it reports Ready.
```

Comment:
I think I'm missing something here - where is the 5s delay happening?  And what is meant here by the sidecar reporting Ready?

Reply:
That was me copy/pasting another example without deleting the comments.  It isn't really waiting for anything to be ready.